astropy/modeling/tests/test_separable.py

```diff
@@ -0,0 +1,107 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Test separability of models.
+
+"""
+
+from astropy.modeling import models
```

Comment:
Why is this using absolute import even though you have relative import below? Why not just have all `astropy` imports be relative imports here?